### ISSUE TITLE
error when using zod with skypack

### ISSUE THREAD

**Issue by yqrashawn** [colinhacks/zod#309]
![image](https://user-images.githubusercontent.com/15090582/106842184-7feddd00-66de-11eb-887d-78338c7be7bd.png)
same error if runs in browser
```html
<!DOCTYPE html>
<html lang="en">
  <body>
    <script type="module">
      import z from 'https://cdn.skypack.dev/zod';
      z.string().parse('fish')
    </script>
    <script type="module">
      import * as z from 'https://cdn.skypack.dev/zod';
      z.string().parse('fish')
    </script>
  </body>
</html>
```

**Comment by jacksteamdev** [comment]
This is likely tied to #175

Snowpack is based on ESM, and zod doesn't currently play nice with ESM.

**Comment by colinhacks** [comment]
@yqrashawn @jacksteamdev I'm working now to refactor Zod to eliminate circular imports (the root issue of the ESM incompatibility). Zod 3 should be available as an ESM module within the week.

**Comment by colinhacks** [comment]
I just released the newest v3 alpha (`zod@3.0.0-alpha.7`) which includes ES module support. 

@yqrashawn can you try `https://cdn.skypack.dev/zod@3.0.0-alpha.7` and let me know if it works?

**Comment by yqrashawn** [comment]
> I just released the newest v3 alpha (`zod@3.0.0-alpha.7`) which includes ES module support.
> 
> @yqrashawn can you try `https://cdn.skypack.dev/zod@3.0.0-alpha.7` and let me know if it works?

This works fine in deno and browser now.
